tests/core/data/test_excel_data_node.py

```diff
@@ -320,12 +352,8 @@ def test_read_multi_sheet_with_header(self):
         excel_data_node_as_pandas_no_sheet_name = ExcelDataNode("bar", Scope.PIPELINE, properties={"path": path})
 
         data_pandas_no_sheet_name = excel_data_node_as_pandas_no_sheet_name.read()
-        assert isinstance(data_pandas_no_sheet_name, Dict)
-        assert len(data_pandas_no_sheet_name) == 2
-        assert data_pandas.keys() == data_pandas_no_sheet_name.keys()
-
-        for sheet_name in sheet_names:
-            assert data_pandas[sheet_name].equals(data_pandas_no_sheet_name[sheet_name])
+        assert isinstance(data_pandas_no_sheet_name, pd.DataFrame)
```

Comment:
Here, the default behavior of pandas when reading an Excel file is that if nothing is provided for the sheet_name parameter, it will try to load "Sheet1" with "Sheet1" is the default value.
In our case, if no sheet_name is provided, we will pass None to the sheet_name parameter. This will load all sheets in the Excel file.
This fix will have a side effect, it stays true to the default behavior of pandas, but if user passes in None and expect it to return all sheets, it will go back to the default and only returns "Sheet1"